### ISSUE TITLE
[Bug]Red magic stones not generating

### ISSUE THREAD

**Issue by LegacyHero25** [Stereowalker/C.O.M.B.A.T#62]
Hello, I am having great trouble finding any of the required Red magic stone needed for upgrading wands as I have been playing on a custom modpack for quite some time now and I am near endgame and have explored many caves and made several mines looking for the Red magic stones but can't find any, this has led me to believe it is one of two reasons.
1) There is either an error with the ore gen as other ores such as tridox and copper spawn however this does not.
2) There is also the possibility that another mod is conflicting with its ore directories and causing them not to spawn.

The versions I am using are as follows:
**Minecraft:**
1.16.1
**Minecraft Forge:**
release-1.16.1-forge 32.0.108
**C.O.M.B.A.T mod:**
C.O.M.B.A.T-1.16.1-5.2.1
**UnionLib:**
UnionLib-1.16.1-2.2.0

Even if you can't fix this issue you can still find a possible mod that conflicts with yours

I have attached a list of my mods that I have been using below:
**P.s** sorry for it only being screenshots I would have just done the file with the mods but it doesn't accept it
[custom modpack mods.docx](https://github.com/Stereowalker/C.O.M.B.A.T/files/5192608/custom.modpack.mods.docx)

**Comment by Stereowalker** [comment]
They don't spawn as ores do, but rather as a structure underground. It spawns at around y50 to y20 and you can use the ```
/locate combat:magic_stone_deposit
``` command to find it.

**Comment by LegacyHero25** [comment]
Thank you for your response as this has helped me a lot because in the wikis information about it it did not mention what Y level it was. I also noticed that I could see diamonds in the background and this confused me into thinking it was at diamond level which is around Y level 14. Also when I ran the command it did show up however I noticed it was quite far away and was wondering on the spawn rate for the structure for example is it 1 in every 5 chunks or 1 in every 20 chunks?. You don't have to respond to this but if you did it would be very appreciated

**Note:**
If you don't respond to this in a week or so I will just close this issue page. Thank you anyway.
